docs/operations.rst

```diff
@@ -66,7 +66,7 @@ An optimization problem using only these functions can be solved by any LP solve
 +---------------------------+----------------------------+------------+---------------+---------------------------------+
 |:code:`dot(x,y)`           | :math:`\sum_i x_i y_i`     | affine     |increasing     | PR: one argument is constant    |
 +---------------------------+----------------------------+------------+---------------+---------------------------------+
-|:code:`kron(x,y)`          | Kronecker product          | affine     |increasing     | PR: one argument is constant    |
+|:code:`kron(x,y)`          | Kronecker product          | affine     |increasing     | PR: First argument is constant  |
```

Comment:
Why can't we just check to make sure at least one argument is constant, like we do for multiplication? (also: consistent capitalization says "First" should be "first".)